site/docs/guidelines/tables-lists.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
+tags: ["Data Tables", "Tabular Data", "Cards", "Tiles", "Rows", "Grids"]
+inComparingSection: true
+---
+
+- **Tables also known as:** Data Tables, Tabular Data
+- **Lists also known as:** Cards, Tiles, Rows, Grids
+
+## Need to know
+### Tables
+- Tables are for presenting information which gives the user the flexibility to sort the data to find their own patterns
+- Tables must include column headers for sorting
+
+### Lists
+- Lists are for presenting grouped information into scannable patterns. The content within a list can be stacked to create hierarchy within the data.
+- Lists are sortable by a ‘Sorted by’ Select, and do not have header labels
```

Comment:
Maybe "column header labels" to make it clear we're not talking about the header header